scope.go

```diff
@@ -744,7 +744,11 @@ func newCluster(c config.Cluster) (*cluster, error) {
 		clusterUsers[cu.Name] = newClusterUser(cu)
 	}
 
-	heartBeat := newHeartBeat(c.HeartBeat, c.ClusterUsers[0])
+	heartBeatClusterUser := c.ClusterUsers[0]
```

Comment:
This code is duplicated. Maybe it can be moved to newHeartBeat? Moreover, should we check if transparent user is allowed?

Reply:
Yes, it is replicated in a test. Actually it is was tests are needed for.
Moving into newHeartBeat  ... well, possible but not sure.